### PR TITLE
searchfragment + radiobuttons

### PR DESCRIPTION
**Pull request by invindus**
imageimage

### CODE REVIEW

**Review by sepniojr — APPROVED** [review]

**Review by thomassuu — APPROVED** [review]

File: code/app/src/main/java/com/example/qrcodesfornoobs/Fragment/SearchFragment.java [hunk 1]
```diff
@@ -1,66 +1,176 @@
...
+
+            @Override
+            public boolean onQueryTextChange(String newText) {
+                searchList = new ArrayList<>();
+
+                if(newText.length() > 0){
+                    for(int i = 0; i < dataList.size(); i++){
+                        if(dataList.get(i).toUpperCase().contains(newText.toUpperCase())){
+                            String str = dataList.get(i);
+                            searchList.add(str);
```
Comment (guerrapavez) [inline code/app/src/main/java/com/example/qrcodesfornoobs/Fragment/SearchFragment.java:145]: use firebase query?

**Review by thomassuu — COMMENTED** [review]
crashing...

**Review by thomassuu — CHANGES_REQUESTED** [review]
please check your branch again as it's crashing on my side

**Comment by invindus** [comment]
rewriting SearchFragment to implement firebase queries

**Pull request closed without merging**